qark/plugins/manifest/debuggable.py

```diff
@@ -30,9 +32,12 @@ def run(self, file_object):
         for application in application_sections:
             try:
                 if application.attributes["android:debuggable"].value.lower() == "true":
-                    self.issues.append(Vulnerability(category=self.category, severity=self.severity,
-                                                     issue_name=self.issue_name, description=self.description,
-                                                     file_object=file_object))
+                    self.issues.add(Vulnerability(category=self.category, severity=self.severity,
+                                                  issue_name=self.issue_name, description=self.description,
+                                                  file_object=manifest_xml))
```

Comment:
I thought file_object was meant to be the filepath?

Reply:
Nice catch, fixed.